.github/workflows/ci.yml

```
@@ -217,6 +217,31 @@ jobs:
           build\${{matrix.buildType}}\run-test262.exe -c tests.conf
           build\${{matrix.buildType}}\function_source.exe
 
+  windows-msvc-vs2019:
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [x64, Win32]
+        buildType: [Debug, Release]
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          cmake -B build -DBUILD_EXAMPLES=ON -G "Visual Studio 16 2019" -A ${{matrix.arch}}
+          cmake --build build --config ${{matrix.buildType}}
```

```suggestion
          cmake --build build --config ${{matrix.buildType}} --target qjs_exe
          cmake --build build --config ${{matrix.buildType}} --target function_source
          cmake --build build --config ${{matrix.buildType}} --target fib
          cmake --build build --config ${{matrix.buildType}} --target point
```
i.e., omit run-test262; 2019 doesn't seem to suport `_Thread_local`